main.go

```diff
@@ -143,6 +145,14 @@ func main() {
 	}
 }
 
+func handleRootRequest( res http.ResponseWriter, r *http.Request) {
+	if len(mainCfg.Listen.RootFallback) == 0 {
```

Comment:
`RootFallback` is a `string` type variable. `if mainCfg.Listen.RootFallback == "" {` no need for length checking: We are in a strongly typed language and it cannot get anything different than a string. So the default is `""`.